crap-advisor-frontend/src/utils/index.ts

```diff
@@ -1,25 +1,35 @@
-import {Location, Primitive} from "../app/typings";
-import {isNil} from 'lodash';
-import {LatLngExpression} from "leaflet";
+import { LatLngExpression } from 'leaflet';
+import { isNil } from 'lodash';
 
-export const params2form = (
-    payload: Record<string, Primitive | Primitive[]>
-): string =>
-    Object.keys(payload)
-        .filter((key) => !isNil(payload[key]))
-        .map((key) => {
-            if (Array.isArray(payload[key])) {
-                return (payload[key] as Primitive[])
-                    .filter((item) => !isNil(item))
-                    .map(
-                        (item) => `${encodeURIComponent(key)}=${encodeURIComponent(item)}`
-                    )
-                    .join('&');
-            }
-            return `${encodeURIComponent(key)}=${encodeURIComponent(
-                payload[key] as Primitive
-            )}`;
-        })
-        .join('&');
+import { Location, Primitive } from 'app/typings';
 
-export const toLatLngExpression = (location: Location): LatLngExpression => [location.latitude, location.longitude]
\ No newline at end of file
+export const params2form = (payload: Record<string, Primitive | Primitive[]>): string =>
+  Object.keys(payload)
+    .filter((key) => !isNil(payload[key]))
+    .map((key) => {
+      if (Array.isArray(payload[key])) {
+        return (payload[key] as Primitive[])
+          .filter((item) => !isNil(item))
+          .map((item) => `${encodeURIComponent(key)}=${encodeURIComponent(item)}`)
+          .join('&');
+      }
+      return `${encodeURIComponent(key)}=${encodeURIComponent(
+        payload[key] as Primitive
+      )}`;
+    })
+    .join('&');
+
+export const toLatLngExpression = (location: Location): LatLngExpression => [
+  location.latitude,
+  location.longitude,
+];
+
+export const getRestroomIdFromURL = () => {
```

Comment:
omg

Reply:
Not a good function, but in case like that i can call it "ok solution"

Reply:
we need only 'great solutions' in our code